pkl-codegen-java/src/test/kotlin/org/pkl/codegen/java/JavaCodeGeneratorTest.kt

```diff
@@ -127,6 +129,9 @@ class JavaCodeGeneratorTest {
     }
   }
 
+  @TempDir
+  lateinit var tempDir: Path
```

Comment:
Two reasons:
1. `tempDir` is used by helper methods below, which makes test code simpler compared to constant passing of `tempDir` as an argument.
2. Removed repetition because this parameter was present on lots of test methods.

In general, there is not much difference between a field and a method parameter for this use case. JUnit Jupiter always creates a fresh instance of the test class for each test, and it always populates the marked fields before the tests are run, so defining temp dir as a variable allows to simplify some of the helper methods logic and reduce duplication.